lib/quby/questionnaires/entities/questions/date_question.rb

```diff
@@ -28,8 +28,8 @@ def day_key
             (@day_key || "#{key}_dd").to_s
           end
 
-          def input_keys
-            [day_key.to_sym, month_key.to_sym, year_key.to_sym]
+          def claimed_keys
```

Comment:
Ja, de input_keys voor vragen zonder opties/subvragen is hetzelfde als de answer_keys.

Hier is ook een test voor voor elk type. Dus gaat goed.